pages/api/captcha.ts

```diff
@@ -0,0 +1,23 @@
+import { NextApiResponse, NextApiRequest } from 'next'
+import axios from 'axios'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const { token } = req.body
+
+    const { data } = await axios({
+      url: 'https://www.google.com/recaptcha/api/siteverify',
+      method: 'post',
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      data: `secret=${process.env.NEXT_PUBLIC_RECAPTCHA_SECRET}&response=${token}`,
+    })
+
+    if (data && data.success) {
+      res.status(200).json({ status: true })
```

Comment:
Is anything reading the `status: true`? I don't see it referred to in the rest of the code.